rear-panel/startup/application-startup/module-startup.S

```diff
@@ -0,0 +1,207 @@
+/**
+  ******************************************************************************
+  * @file   module-startup.S
```

Comment:
This file can go, it's a holdover from an earlier rev of the startup app. Should honestly get deleted from the modules repo too.